Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerSuggestionsViewController.swift

```diff
@@ -6,26 +6,28 @@ import StreamChat
 import UIKit
 
 public protocol SuggestionsViewControllerPresenter: class {
```

Comment:
I don't think we need this abstraction at all. Why not _just_ present the controller? :)

Reply:
Deleted :)